specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/preview/2020-09-01-preview/iotdps.json

```diff
@@ -1947,6 +1951,16 @@
         }
       }
     },
+    "KEKIdentity": {
```

Comment:
What does `KEKIndentity` means? For SDK automation, the definition name would be generated into class name. Not sure if it could have more readable name.

Reply:
KEK stands for Key Encryption Key a standard nomeclature as it pertains to customer managed keys. KEKIdentity is the identity used to access the KEK. Is you suggestion to describe this in full?